Update README.md

azarboon opened the pull request:
Before diving into any of the patterns, readers should be reminded of two fundamental laws in software architecture:

1.Everything is a trade-ff
2."Why is more important than the how"

So, readers face the nuances and reality of these patterns from the beginning. These two laws are coined by two thought leaders in software architecture: Mark Richards and Neal Ford. They have explained these two laws in various conference talks and books. For example, here you can read about these two laws here:

https://www.infoq.com/podcasts/software-architecture-hard-parts/


Also, here is a book for reference:
https://a.co/d/fKOodW9

- [x] Read [CONTRIBUTING.md](https://github.com/ochococo/Design-Patterns-In-Swift/blob/master/CONTRIBUTING.md])
- [ ] Only edited files inside `source` folder (IMPORTANT) and commited them with a meaningful message
- [ ] Ran `generate-playground.sh`, no errors
- [ ] Opened playground, it worked fine
- [ ] Did not commit the changes caused by `generate-playground.sh`
- [ ] Linked to and/or created issue
- [x] Added a description to PR

azarboon committed:
Update README.md

Before diving into any of the patterns, readers should be reminded of two fundamental laws in software architecture:

1.Everything is a trade-ff
2."Why is more important than the how"

So, readers face the nuances and reality of these patterns from the beginning. These two laws are coined by two thought leaders in software architecture: Mark Richards and Neal Ford. They have explained these two laws in various conference talks and books. For example, here you can read about these two laws here:

https://www.infoq.com/podcasts/software-architecture-hard-parts/


Also, here is a book for reference:
https://a.co/d/fKOodW9

ochococo commented:
@azarboon Hi, thanks for contributing. Agreed with the comment, don't mind adding that. 

Please follow instruction to only edit files in `source` folder, thank you 🙏

azarboon committed:
changed source file

azarboon commented:
> @azarboon Hi, thanks for contributing. Agreed with the comment, don't mind adding that.
> 
> Please follow instruction to only edit files in `source` folder, thank you 🙏

Thanks. I followed your suggestion and changed the source file.

FYI, I'm editing files on my browser. In case I added / removed unnecessary spaces, kindly please fix them.

ochococo commented on source/contentsReadme.md:
```diff
@@ -1,3 +1,4 @@
+Remember that each pattern has its own trade-offs. And you need to pay attention more to why you're choosing a certain pattern than to how to implement it.
```
I agree. This line would be approved.

ochococo commented on source/contents.md:
```diff
@@ -1,4 +1,3 @@
-
```
Not needed change.

ochococo commented on README.md:
````diff
@@ -17,8 +17,6 @@ A short cheat-sheet with Xcode 10.2 Playground ([Design-Patterns.playground.zip]
 ```swift
 print("Welcome!")
 ```
-
````
Not needed change, same with next line

ochococo reviewed (changes requested):
Thanks for contributing, please remove changes that are not relevant, thank you
